doc/syscalls/90001_screen.md

```diff
@@ -8,6 +8,14 @@ driver number: 0x90001
 
 The screen driver allows the process to write data to a framebuffer of a screen.
 
+This syscall driver is designed for single-client usage, for the sake of simplicity, and because a single client covers a significant portion of display use cases.
```

Comment:
```suggestion
This syscall driver is designed for single-client usage, for the sake of simplicity, and because a single 
client covers a significant portion of display use cases.
```

Please use shorter lines